src/sidemenu/BoardsMenuSection.tsx

```diff
@@ -17,7 +17,7 @@ const BoardsMenuSection: React.FC<BoardsMenuSectionProps> = (props) => {
   } = props as LoadingSectionProps;
   const isEmpty = !loading && (!boards || boards.length == 0);
   return (
-    <div className="boardSection">
+    <section className="boardSection">
```

Comment:
Since you're doing this change, would you mind removing `boardSection` from the classnames? examples:

`. boardSection {` => `section {`

`boardSection-title` => `.title`

(etc.) I had commissioned this UI to someone and the style is inconsistent with the one I use everywhere else.

Reply:
Sure